src/utils/request/onUnhandledRequest.ts

```diff
@@ -126,7 +137,10 @@ export function onUnhandledRequest(
     return
   }
 
-  const parsedGraphQLQuery = parseGraphQLRequest(request)
+  // TODO: comment and exlain why we don't want to ignore the error if parsing fails here
```

Comment:
We actually _want_ to ignore any errors originating during this parsing. I'll provide some explanation below.

```ts
/**
 * @note Ignore any parsing exceptions.
 * At this step, the library cannot guarantee that the unhandled request
 * is a valid GraphQL requests. Since the point of this function is to
 * provide smart suggestions and not ensure a request's type validity,
 * the exception during GraphQL parsing only means that this request
 * is not a valid GraphQL request and shouldn't be suggested as such.
 */
```